yarn.lock

```diff
@@ -144,7 +144,7 @@
   dependencies:
     "@babel/types" "^7.0.0"
 
-"@babel/helper-module-transforms@^7.1.0", "@babel/helper-module-transforms@^7.4.4":
+"@babel/helper-module-transforms@^7.1.0":
```

Comment:
Please remove this file from your PR (eventually)...

Reply:
sure